docs/DeveloperGuide.md

```diff
@@ -762,22 +825,46 @@ testers are expected to do more *exploratory* testing.
     1. Prerequisites: List all companies using the `list` command. Multiple companies in the list.
 
     2. Test case: `delete 1`<br>
-       Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+       Expected: First contact is deleted from the list. Details of the deleted company shown in the message box.
 
     3. Test case: `delete 0`<br>
-       Expected: No company is deleted. Error details shown in the status message. Status bar remains the same.
+       Expected: No company is deleted. Error details shown in the message box.
 
     4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
        Expected: Similar to previous.
 
-2. _{ more test cases …​ }_
+2. Deleting a company when some companies are being shown
+    1. Prerequisites: Filter the list of companies by their application status (eg. Pending Application) using the
+       `filter s/pa` command. At least one company in the filtered list.
+    2. Test case: `delete 1`<br>
+       Expected: First company in the filtered list is deleted from the list. Details of the deleted company shown in the
+       message box.
+    3. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Viewing a Company
+
+1. Viewing a company while all the companies are being shown
+    1. Prerequisites: List all companies using the `list` command. Multiple companies in the list.
+    2. Test case: `view 1`<br>
+       Expected: First company is shown in the company detail panel.
+    3. Test case: `view 0`<br>
+       Expected: No company is shown in the company detail panel. Error details shown in the message box.
+    4. Other incorrect view commands to try: `view`, `view x`, `...` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
```

Comment:
not sure how we want to standardise with my pr as well. 

are we doing it very detailed?